plugins/modules/packaging/os/urpmi.py

```diff
@@ -189,13 +194,13 @@ def root_option(root):
 def main():
     module = AnsibleModule(
         argument_spec=dict(
-            state=dict(type='str', default='installed',
+            state=dict(type='str', default='present',
                        choices=['absent', 'installed', 'present', 'removed']),
             update_cache=dict(type='bool', default=False, aliases=['update-cache']),
             force=dict(type='bool', default=True),
             no_recommends=dict(type='bool', default=True, aliases=['no-recommends']),
-            name=dict(type='list', required=True, aliases=['package', 'pkg']),
-            root=dict(type='str', aliases=['installroot']),
+            name=dict(type='list', elements='str', required=True, aliases=['package', 'pkg']),
+            root=dict(type='str', aliases=['installroot'], default='/'),
```

Comment:
```suggestion
            root=dict(type='str', aliases=['installroot']),
```
Please undo this change (and remove `default` in the option docs). Even if `default='/'` would be correct here (it would be if `--root=/` in the CLI for urpmi is equivalent to not specifying `--root`), it's better to mention such defaults in the documentation's description instead of actually setting them as default.

(It could for example be that urpmi allows to change the default via the config file or a env variable, in which case this default would be a breaking change for users having defined another root via the config file or env variable.)